apps/minifront/src/components/swap/layout.tsx

```diff
@@ -6,13 +6,18 @@ import { UnclaimedSwaps } from './unclaimed-swaps';
 
 export const SwapLayout = () => {
   return (
-    <div className='grid gap-6 md:grid-cols-2 md:gap-4 xl:grid-cols-3 xl:gap-5'>
-      <UnclaimedSwaps />
-      <Card gradient className='order-3 row-span-2 flex-1 p-5 md:order-1 md:p-4 xl:p-5'>
-        <SwapForm />
-      </Card>
-      <div className='md:order-2'>
-        <EduInfoCard src='./swap-icon.svg' label='Shielded Swap' content={EduPanel.SWAP} />
+    <div className='mx-auto max-w-[1276px]'>
```

Comment:
I've noticed this wrapper is also used in several other layout components. Could you extract this to a `<MaxWidthWrapper />` or something, so that we ensure consistent layout?